File: Java/src/src/game_logic/AdventureCard.java
package src.game_logic;

public class AdventureCard extends Card{
	protected int battlePoints;
	
	public AdventureCard(String name) {
		super(name);
	}
	
	public AdventureCard(String name, int battlePoints) {
		super(name);
		this.battlePoints = battlePoints;
	}
	
	public int getBattlePoints() {
		return this.battlePoints;
	}
}

File: Unity/QuestOfTheRoundTable/Assets/Scripts/Objects/AdventureCard.cs
namespace Card
{
    public enum AdventureType
    {
        Foe, Weapon, Ally, Armour, Test
    }
    public class AdventureCard : Card
    {
        public int BattlePoints { get; set; }
        public int NamedBattlePoints { get; set; }
        public AdventureCard(string name): base(name)
        {
        }
        public AdventureCard(string name, int bp) : base(name)
        {
            BattlePoints = bp;
        }
        public AdventureCard(string name, int bp, int nbp) : base(name)
        {
            BattlePoints = bp;
            NamedBattlePoints = nbp;
        }
    }
}

